resources/dicts/events/injury/forest.json

```diff
@@ -284,40 +284,5 @@
                 "injuries": [ "headache" ]
             }
         ]
-    },
-    {
```

Comment:
was this intentionally removed? i don't think i saw it added to a different file

Reply:
Yeah - it got moved to a general injury :) No point in only being edible in a forest lol